package-lock.json

```diff
@@ -21082,175 +20595,469 @@
-      "dev": true,
-      "license": "ISC",
+    "node_modules/mongodb-mcp-server/node_modules/@mongosh/errors": {
+      "version": "2.4.2",
+      "resolved": "https://registry.npmjs.org/@mongosh/errors/-/errors-2.4.2.tgz",
+      "integrity": "sha512-p+LOHVj/VIt6cpJY1AvDmG/QLP7WZQ7q+32paU6qxjXaoC0kMqmemaqK5cUj8JWod1VEv9/Ol4T6OfWTwPG20A==",
+      "license": "Apache-2.0",
       "engines": {
-        "node": ">=10"
+        "node": ">=14.15.1"
       }
     },
-    "node_modules/mongodb": {
-      "version": "6.19.0",
-      "resolved": "https://registry.npmjs.org/mongodb/-/mongodb-6.19.0.tgz",
-      "integrity": "sha512-H3GtYujOJdeKIMLKBT9PwlDhGrQfplABNF1G904w6r5ZXKWyv77aB0X9B+rhmaAwjtllHzaEkvi9mkGVZxs2Bw==",
+    "node_modules/mongodb-mcp-server/node_modules/@mongosh/service-provider-core": {
+      "version": "3.3.5",
+      "resolved": "https://registry.npmjs.org/@mongosh/service-provider-core/-/service-provider-core-3.3.5.tgz",
+      "integrity": "sha512-aLj9Yajf96lSdERo8faP/h6A5mxt4NA3YWHWvZuhop4PQLiS+OdwrqdHFdlTISkAfv+7xZtcbS4FKFfTkoK85A==",
       "license": "Apache-2.0",
       "dependencies": {
-        "@mongodb-js/saslprep": "^1.1.9",
-        "bson": "^6.10.4",
-        "mongodb-connection-string-url": "^3.0.0"
+        "@aws-sdk/credential-providers": "^3.525.0",
+        "@mongosh/errors": "2.4.2",
+        "bson": "^6.10.3",
+        "mongodb": "^6.16.0",
+        "mongodb-build-info": "^1.7.2",
+        "mongodb-connection-string-url": "^3.0.1"
       },
       "engines": {
-        "node": ">=16.20.1"
-      },
-      "peerDependencies": {
-        "@aws-sdk/credential-providers": "^3.188.0",
-        "@mongodb-js/zstd": "^1.1.0 || ^2.0.0",
-        "gcp-metadata": "^5.2.0",
-        "kerberos": "^2.0.1",
-        "mongodb-client-encryption": ">=6.0.0 <7",
-        "snappy": "^7.3.2",
-        "socks": "^2.7.1"
+        "node": ">=14.15.1"
       },
-      "peerDependenciesMeta": {
-        "@aws-sdk/credential-providers": {
-          "optional": true
-        },
-        "@mongodb-js/zstd": {
-          "optional": true
-        },
-        "gcp-metadata": {
-          "optional": true
-        },
-        "kerberos": {
-          "optional": true
-        },
-        "mongodb-client-encryption": {
-          "optional": true
-        },
-        "snappy": {
-          "optional": true
-        },
-        "socks": {
-          "optional": true
-        }
+      "optionalDependencies": {
+        "mongodb-client-encryption": "^6.3.0"
       }
     },
-    "node_modules/mongodb-build-info": {
-      "version": "1.7.2",
-      "resolved": "https://registry.npmjs.org/mongodb-build-info/-/mongodb-build-info-1.7.2.tgz",
-      "integrity": "sha512-eoLFZvCIjcwijYJdxvYupj1c+55VAVm0o4gBJjrcDxxmmpm+bC4Ix9ayZbyhQdVXDZAGDi03NA0GghXjBVXnxg==",
+    "node_modules/mongodb-mcp-server/node_modules/@mongosh/service-provider-node-driver": {
```

Comment:
^ This is probably the reason for the test failure – you basically always want `@mongosh/service-provider-node-driver` to be hoisted without duplicates